tests/integration/internal/stepfuncs/logs.go

```diff
@@ -42,3 +42,9 @@ func GenerateLogsWithDeployment(
 		return ctx
 	}
 }
+
+func DeployPodGeneratingMultilineLogs() features.Func {
+	return func(ctx context.Context, t *testing.T, envConf *envconf.Config) context.Context {
+		return ctx
+	}
+}
```

Comment:
🤔

Reply:
Thanks for catching this, it is no longer needed.